servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -737,11 +737,7 @@ protected void handleSubscribe(final Subscriber subscriber) {
 
         void nextError(final Throwable t) {
             seenError = true;
-            final CompletableSource.Processor oldProcessor = processor;
```

Comment:
@idelpivnitskiy - can you confirm I didn't miss anything here? This seems unnecessary now as the underlying `CompletableSource.Processor` will flush its stack when it completes and new listeners will be notified of the terminal state.

Reply:
moved to https://github.com/apple/servicetalk/pull/2115